projects/pulsar_adc_pmdz/zed/system_bd.tcl

```diff
@@ -0,0 +1,24 @@
+###############################################################################
+## Copyright (C) 2021-2023 Analog Devices, Inc. All rights reserved.
+### SPDX short identifier: ADIBSD
+###############################################################################
+
+source $ad_hdl_dir/projects/common/zed/zed_system_bd.tcl
+source $ad_hdl_dir/projects/scripts/adi_pd.tcl
+
+# specify the spi reference clock frequency in MHz
+set spi_clk_ref_frequency 160
```

Comment:
I think that this variable is no longer used in the current implementation.

Reply:
I removed it.